src/main/java/linkedList/Node.java

```diff
@@ -0,0 +1,8 @@
+package linkedList;
+
+public class Node<T> {
+    T data;
+    Node next;
+    Node prev;
```

Comment:
LinkedList can be either one-directioned or two-directioned:
* a -> b -> c
* a <-> b <-> c
Was it really required to implement two-directioned?

Reply:
It wasn't specified in the task =)

Reply:
yeah, I see, but one-direction list is a bit simplier :)